packages/x-charts-pro/src/internals/plugins/useChartProExport/print.ts

```diff
@@ -0,0 +1,91 @@
+import ownerDocument from '@mui/utils/ownerDocument';
+import { ChartPrintExportOptions } from './useChartProExport.types';
+
+export function printChart(
```

Comment:
This code is practically a copy of [Data Grid's export functionality](https://github.com/mui/mui-x/blob/d8b9dfbd2b30b33ac250d4d74ee736ed1bbed87c/packages/x-data-grid/src/hooks/features/export/useGridPrintExport.tsx#L354).